overlord/devicestate/firstboot_test.go

```diff
@@ -165,6 +165,16 @@ type firstBoot16Suite struct {
 
 var _ = Suite(&firstBoot16Suite{})
 
+func mockSystemctlShowForMountUnit(name, active, needReload string) []byte {
+	return []byte(fmt.Sprintf(`Id=%s
+Names=%s
+Type=squashfs
+UnitFileState=
+NeedDaemonReload=%s
+ActiveState=%s
+`, name, name, active, needReload))
```

Comment:
The last two parameters are swapped.

Reply:
this has been dropped, along with the other one and changes in the affected tests